contracts/modules/capital/Pool.sol

```diff
@@ -731,6 +735,22 @@ contract Pool is IPool, MasterAware, ReentrancyGuard {
     }
 
     if (code == "PRC_FEED") {
+
+      uint coverAssetsCount = coverAssets.length;
+
+      for (uint i = 0; i < coverAssetsCount; i++) {
+        Asset memory asset = coverAssets[i];
+
+        (address aggregator, ) = priceFeedOracle.assets(asset.assetAddress);
```

Comment:
This call seems redundant if the asset is ETH. Move this call inside the if condition below.